Makefile

```diff
@@ -32,15 +31,16 @@ endif
 
 CFLAGS := $(CFLAGS) -std=c11 $(SHARED_FLAGS) -Wmissing-prototypes
 CXXFLAGS := $(CXXFLAGS) -std=c++17 $(SHARED_FLAGS)
-LDFLAGS := $(LDFLAGS) -Wl,--as-needed,-z,defs,-z,relro,-z,now,-z,nodlopen,-z,text
```

Comment:
There _has_ to be an "Apple Clang" alternative to `-z,arg` here or whatever. Might wanna look into that.